lightning/src/routing/network_graph.rs

```diff
@@ -259,6 +260,8 @@ pub struct ChannelInfo {
 	/// Everything else is useful only for sending out for initial routing sync.
 	/// Not stored if contains excess data to prevent DoS.
 	pub announcement_message: Option<msgs::ChannelAnnouncement>,
+	/// The minimum fee the user is willing to pay to avoid using a channel
+	pub minimum_fee_penalty: Option<u32>,
```

Comment:
One nit: the naming should be consistent in terms of `fee_penalty` vs `penalty_fee`.